wls-infomanagement-service/sso-config/sso-client.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
File kann auch gelöscht werden. Der Client in Keycloak im Docker wird anders konfiguriert.

Reply:
Gelöscht mit f9a83a9